pkgs/top-level/release-lib.nix

```diff
@@ -43,6 +46,11 @@ rec {
   pkgs_x86_64_cygwin = allPackages { system = "x86_64-cygwin"; };
 
 
+  assertTrue = bool:
+    if bool
+    then pkgs.runCommand "evaluated-to-true" {} "touch $out"
+    else pkgs.runCommand "evaluated-to-false" {} "false";
```

Comment:
This is basically what the module tests are doing, they basically spawn multiple Nix expressions tests within one job which contains all module tests.